docs/guides/errors.md

```diff
@@ -224,6 +224,36 @@ paths:
 
 `curl -X POST http://localhost:4010/ -A 'Accept: text/plain'`
 
+### NO_SUCCESS_RESPONSE_DEFINED
+
+**Message: No 2** response defined, cannot mock\*\*
+**Returned Status Code: `500`**
+**Explanation:** This error occurs when the current request has matched a corresponding HTTP Operation and has passed all the validations, but there's no successful response (200-299) that could be returned.
+
+##### Example
+
+```yaml
+openapi: '3.0.1'
+paths:
+  /:
+    get:
+      responses:
+        300:
+          content:
+            text/plain:
+              schema:
+                type: object
+                properties:
+                  name:
+                    type: string
+                    example: Clark
+                  surname:
+                    type: string
+                    example: Kent
```

Comment:
How about something simpler? We just want to emphasize that there is no 2xx response.

```suggestion
      responses:
        300:
          description: 'Desc'
```

Reply:
Yes!